src/libsync/propagateupload.cpp

```diff
@@ -605,6 +605,22 @@ QMap<QByteArray, QByteArray> PropagateUploadFileCommon::headers()
         //  csync_owncloud.c's owncloud_file_id always strips the quotes.
         headers["If-Match"] = '"' + _item->_etag + '"';
     }
+
+    // Set up a conflict file header pointing to the original file
+    if (propagator()->account()->capabilities().uploadConflictFiles()
```

Comment:
Why do we need to check the capabilities here?

Reply:
True, unnecessary